tests/jquery.flot.touchNavigate.Test.js

```diff
@@ -69,95 +69,6 @@ describe("flot touch navigate plugin", function () {
             expect(yaxis.max).toBeCloseTo(initialYmax - (initialYmax - midPointCoords.y) * (1 - 1/amount), 6);
         });
 
-        it('for multiple touches should take into account just the first two of them and zoom',function() {
```

Comment:
One think I didn't like about pinch with multiple touches was that the touche order is not guaranteed. Strange things could happen if you don't take the order into account. Removing this functionality is removing the need to handle extra complexity.